build/buildBuildImages.sh

```diff
@@ -151,6 +151,9 @@ echo
 echo "List of images tagged (from '$ACR_BUILD_IMAGES_ARTIFACTS_FILE'):"
 cat $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 
+echo
+showDockerImageSizes
```

Comment:
is this permanent or was for debugging?

Reply:
Yes I wanted it to be there since it felt useful to see those details